pom.xml

```diff
@@ -201,7 +201,7 @@
       <dependency>
         <groupId>org.ow2.asm</groupId>
         <artifactId>asm</artifactId>
-        <version>6.2</version>
+        <version>6.2.1</version>
```

Comment:
I would prefer using now 7.0

Reply:
Sure!

Reply:
Done in 152ba6138f3e8f8b9745f21d7074aa79eea38624